openassessment/xblock/test/test_staff_area.py

```diff
@@ -781,3 +781,44 @@ def _create_submission(item, values, types):
         peer_api.on_start(submission["uuid"])
         workflow_api.create_workflow(submission["uuid"], types)
         return submission
+
+    @scenario('data/self_only_scenario.xml', user_id='Bob')
+    def test_staff_area_student_info_image_submission_names(self, xblock):
+        # Bug fix , In some places files_name as a key is used and in other places files_names is used
```

Comment:
The docstring can be improved, but rest looks good to me, You would also need to update `version` in `setup.py`. 

e.g. 
https://github.com/edx/edx-ora2/pull/1322/files

Reply:
+1 for that

Reply:
Updated versions

Reply:
@AhtishamShahid The docstring format is still incorrect. Please see some examples on edx-platform (https://github.com/edx/edx-platform/pull/22298/). Add the scenario in the docstring to give the clear intent of the test case.